### PR TITLE
Adding akka-diagnostics

### PR DESCRIPTION
**Pull request by franciscolopezsancho**
References https://github.com/akka/akka-diagnostics/issues/19

### CODE REVIEW

File: alpakka-sample-mqtt-to-kafka/project/Dependencies.scala [hunk 1]
```diff
@@ -26,6 +26,6 @@ object Dependencies {
     "com.typesafe.akka" %% "akka-slf4j" % AkkaVersion,
     "ch.qos.logback" % "logback-classic" % "1.2.3",
   // #deps
-    "org.testcontainers" % "kafka" % "1.14.1"
+    "org.testcontainers" % "kafka" % "1.16.0"
```
Comment (franciscolopezsancho) [inline alpakka-sample-mqtt-to-kafka/project/Dependencies.scala:29]: to overcome 
```
11:40:14.358 WARN  [org.testcontainers.utility.ResourceReaper]         Can not connect to Ryuk at localhost:55000
java.net.ConnectException: Connection refused
```

**Comment by franciscolopezsancho** [comment]
Tested a few of them and all good: ConfigChecker is loaded but no warnings are present.

File: alpakka-sample-http-csv-to-kafka/project/Dependencies.scala [hunk 2]
```diff
@@ -4,17 +4,19 @@ object Dependencies {
   val scalaVer = "2.13.7"
   // #dependencies
   val ScalaTestVersion = "3.1.4"
-  val AkkaVersion = "2.6.19"
+  val AkkaVersion = "2.7.0"
   val AkkaHttpVersion = "10.1.12"
   val AlpakkaVersion = "4.0.0"
   val AlpakkaKafkaVersion = "3.0.1"
```
Comment (patriknw) [inline alpakka-sample-http-csv-to-kafka/project/Dependencies.scala:10]: could you update all versions to latest final? You find the versions in https://github.com/lightbend/lightbend-technology-intro-doc/blob/master/docs/modules/getting-help/examples/build.sbt

Comment (franciscolopezsancho) [reply alpakka-sample-http-csv-to-kafka/project/Dependencies.scala:10]: sure

Comment (franciscolopezsancho) [reply alpakka-sample-http-csv-to-kafka/project/Dependencies.scala]: done

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**